templates/views/chapters.html

```diff
@@ -0,0 +1,80 @@
+{% extends "layouts/default.html" %} {% block css %}
+<link rel="stylesheet" type="text/css" href="styles/chapters.css">{% endblock %} {% block head %}{% endblock %} {% block intro %}
+<div id="chapter_banner">
+    <div class="banner-text">
+        <div class="banner-title">
+            <h2>She Code Africa <br> Chapters</h2>
+        </div>
+        <div class="banner-subtitle">
+            <p>
+                You might want to consider starting a SheCodeAfrica Chapter in your campus or city. Not an expert in tech yet? Not a problem (CTA)
+            </p>
+        </div>
+        <div class="banner-btn">
+            <button class="btn sca-btn pink-btn">JOIN A CHAPTER</button>
+            <button class="btn sca-btn trans-btn">LEAD A CHAPTER</button>
```

Comment:
On `:focus`, the text becomes less visible
<img width="669" alt="Screenshot 2020-09-03 at 10 22 22" src="https://user-images.githubusercontent.com/28678648/92097168-ae822780-edcf-11ea-8460-24cc38762d21.png">

Reply:
I  didn't experience this, can you refresh?

Reply:
Still the same. It happens when I click on the button